de-lib/src/main/java/org/iplantc/de/apps/client/presenter/list/AppsListPresenterImpl.java

```diff
@@ -108,79 +87,41 @@
     OntologyServiceFacade ontologyService;
     OntologyUtil ontologyUtil;
     HandlerManager handlerManager;
-    AppsListView gridView, tileView, activeView;
-    CardLayoutContainer cards;
-    AppTypeFilter filter;
+    AppsListView listView;
+    String filter;
     private OntologyHierarchy selectedHierarchy;
     private AppCategory appCategory;
     private App desiredSelectedApp;
+    private String activeView;
+    private List<App> selectedApps = new ArrayList<App>();
```

Comment:
Have save a copy of selection since, `AppsViewToolbarImpl` still needs `App` autobean (not splittable).